resources/monitoring/charts/prometheus-pushgateway/templates/policy.yaml

```diff
@@ -0,0 +1,11 @@
+# Required because Prometheus Operator doesn't have Istio Sidecar
```

Comment:
Please create a kyma-additions folder and put all the additional files that did not exist on the original chart there.